### ISSUE TITLE
Latest updates cause errors

### ISSUE THREAD

**Issue by Plaenkler** [StephaneBour/actions-calver#20]
Since the last updates, all workflows with this action fail. I don't understand the problem straight away.

```text
Unable to process file command 'output' successfully.
Invalid format '* Merge pull request #29 from LaKiS-GbR/dependencies ([50394ec](https://github.com/LaKiS-GbR/***/commit/***)), closes [#29](https://github.com/LaKiS-GbR/***/issues/29)'
```

```yaml
jobs:
  release:
    name: Release
    runs-on: ubuntu-latest
    
    steps:
      - uses: actions/checkout@v4

      - name: Calver Release
        uses: StephaneBour/actions-calver@master
        id: calver
        with:
          pre: false
          release: true
          date_format: "%Y.%V"
          release_branch: main
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

**Comment by pedrograu** [comment]
I am facing the same error also from the last updates, here is my configuration

```
name: Specify release version
        uses: StephaneBour/actions-calver@master
        id: calver
        with:
          date_format: "%Y-%m-%d-%H%M"
          release: true
          release_branch: main
     env:
          GITHUB_TOKEN: ${{ secrets.CALVER }}     
```

**Comment by craciun-st** [comment]
It may be due to the `$(conventional-changelog)` (as `${MESSAGE}`) which `$GITHUB_OUTPUT` does not like.
Maybe it tries to interpret it as Markdown 🤔 (https://github.github.com/gfm/)

Perhaps quoting the text in the variable:
```sh
echo "changelog=\"${MESSAGE}\"" >>$GITHUB_OUTPUT
```
would be a quick & dirty fix to the commit here: https://github.com/StephaneBour/actions-calver/commit/b5019659dee285ff3963e2564b2aa48989df90a4

**Comment by faheel** [comment]
You can pin to a working version for now to resolve this:

```
uses: StephaneBour/actions-calver@v1.6
```

**Comment by miigotu** [comment]
Yes it is indeed the changelog. Unfortunately even if I make it a multi-line output, it still errors then on the changelog. I'll probably just remove that.

Could you try your workflow with `uses: StephaneBour/actions-calver@bugfix/20-changelog` and make sure that works?

**Comment by miigotu** [comment]
I can't merge that PR until I'm on a PC, I need to fix the commit message and remove the changelog from the outputs. After 30+ years of development I'm still getting used to always using conventional commit formats so bear with me lol.

**Comment by craciun-st** [comment]
Can confirm action passes with `uses: StephaneBour/actions-calver@bugfix/20-changelog`

**Comment by miigotu** [comment]
@craciun-st I have a more complete fix in #22 with the changelog output still in, could you try with that branch? I will hopefully set up workflows to test these changes in the near future but until then this is the best I can do.

**Comment by craciun-st** [comment]
It also passes with `actions-calver@fix-outputs` ☑️
